docs/baseband/mark5b/index.rst

```diff
@@ -7,11 +7,11 @@ MARK 5B
 *******
 
 The Mark 5B format is the output format of the Mark 5B disk-based VLBI data
-system.  It is described in its `design specifications <m5bspec>`_ (these
+system.  It is described in its `design specifications`_ (these
```

Comment:
Can one move the parenthetical sentence to the "footnote" too?

Reply:
No, and since (as far as I know) there's no more information there than in the design document, I'm removing the parenthetical.